lib/Client.php

```diff
@@ -134,7 +132,7 @@ public function transaction() {
      */
     public function close() {
         /** @var Promise $promise */
-        $promise = all(promises($this->promisors));
+        $promise = all($this->promisors);
```

Comment:
well… benchmark it, then you'll see? As far as I remember, we discussed dropping it for Amp v2 and it was something like a few percent difference in performance in Aerys.